requirements.txt

```diff
@@ -1 +1,3 @@
+Adafruit-Blinka
 bleak
+janus
```

Comment:
What is janus and why do you need it?

Reply:
I don't need it any more; I forgot to remove it. It is a queue that you can use both from the sync and async side. Originally I was using it to pass the scanning results back, but I restructured things and it became unnecessary.

https://github.com/aio-libs/janus